### PR TITLE
Fractional Knapsack in C++, Java, Python

### PR DESCRIPTION
**Pull request by singh-shreya6**
#587

### CODE REVIEW

**Comment by jainaman224** [comment]
@singh-shreya6 It appears that you are facing issue in squashing. Am I right?

File: Knapsack_Fractional/Knapsack_Fractional.cpp [hunk 1]
```diff
@@ -0,0 +1,62 @@
+/*
+Fractional Knapsack problem is a variation of Knapsack problem where we can choose items by breaking them in fractions.
+The objective is to fill the knapsack with items to get maximum benefit without crossing the weight capacity of the knapsack.
+*/
+#include<bits/stdc++.h>
+using namespace std;
+
+// Function to compute maximum profit obtainable
+double fractionalKnapsackProblem(vector<pair<double,pair<double,double> > > arr, int n, int W)
```
Comment (jainaman224) [inline Knapsack_Fractional/Knapsack_Fractional.cpp]: Use more descriptive variable names than `n` and `W`.

Comment (singh-shreya6) [reply Knapsack_Fractional/Knapsack_Fractional.cpp]: I have made changes to variable names

Comment (singh-shreya6) [reply Knapsack_Fractional/Knapsack_Fractional.cpp]: Yes, I was having trouble with squashing as my branch was not synced with the repo.
Also, I have tried to make the three codes as similar as possible. Logic wise all three are exactly same. In terms of Data Structures, there is difference in implementation as we have STL in C++ which can store pairs. In Java in order to implement same structure I have used Class, whereas in python, list is available which holds tuples.
I have also made changes to the variable names.

**Comment by jainaman224** [comment]
@singh-shreya6 You have to squash commits again. If you are having trouble and GSSoC student. Then you can ask on slack channel else you can ask on gitter.

**Comment by singh-shreya6** [comment]
Squashed commits, please review

**Pull request merged**